### PR TITLE
update all python Readmes to describe dev process without gradle

### PR DESCRIPTION
**Pull request by sherifnada**
## What
Applies changes from #2441 to all python modules. 

## Pre merge checklist
- [ ] update all connector images to 0.1.0 of base images

### CODE REVIEW

File: airbyte-integrations/connector-templates/generator/plopfile.js [hunk 1]
```diff
@@ -39,25 +39,31 @@ module.exports = function (plop) {
     description: 'Generate an Airbyte Source written in Python',
     prompts: [{type: 'input', name: 'name', message: 'Source name, without the "source-" prefix e.g: "google-analytics"'}],
     actions: [
-        {
```
Comment (cgardens) [inline airbyte-integrations/connector-templates/generator/plopfile.js:42]: why is this changing?

**Review by cgardens — APPROVED** [review]
would it make sense to keep both the gradle and the non gradle instructions instead of removing the gradle ones?

**Review by jrhizor — APPROVED** [review]

File: airbyte-integrations/connector-templates/generator/plopfile.js (hunk 1 above)
Comment (sherifnada) [reply airbyte-integrations/connector-templates/generator/plopfile.js:42]: because I forgot to exclude it from the change.

**Review by davinchia — APPROVED** [review]
Curious, how did you apply the changes to all the python modules?

**Comment by sherifnada** [comment]
@davinchia I applied to all Python modules by re-running the generator and overwriting existing READMEs

**Pull request merged**